README.md

```diff
@@ -32,19 +38,24 @@ To run the indexer, simply enter the repository root and run:
 $ cargo run --bin metaplex-indexer
 ```
 
-## Running `rpc`
+## Running HTTP Servers
+
+If port `3000` is already in use the `PORT` environment variable can be used chang the listener port of the servers:
```

Comment:
The server actually uses a command line argument that defaults to pulling from the `PORT` env var, we should probably mention this.  Additionally, all binaries should have a `--help` option we can mention that explains all this as well.